backend/templates/emails/reminder-email.html

```diff
@@ -0,0 +1,84 @@
+{% extends 'emails/base.html' %}
```

Comment:
Really minor nitpick, but could you rename this file into `renewal_reminder.html`?

There is already a `reminder.html` file, which may make it confusing to tell the difference between `reminder` and `reminder-email`. I can rename the `reminder.html` email to `update_reminder.html` and update all of the references later on.

I also chose the underscore instead of the dash since the other template I just created uses the underscore. It doesn't matter which one we choose (`_` or `-`), but we should be consistent in which it is.

Reply:
Got it! Thanks!